blocks/site-details/site-details.css

```diff
@@ -424,11 +424,12 @@ dialog.update-description-dialog button[type="submit"] {
 
 
 /* MARK: Dark alley */
-.site-details.block.dark-alley .button.update-description,
+
+/* .site-details.block.dark-alley .button.update-description,
```

Comment:
Did you restore all these features already ?

Reply:
Just about, yeah. Most of the features work right away by bypassing permission check on the backend if the project is a DA project.

Reply:
In that case you can remove these lines ?